### ISSUE TITLE
Expected array

### ISSUE THREAD

**Issue by andrey-helldar** [laravel/framework#26398]
- Laravel Version: 5.7.12
- PHP Version: 7.2.9
- Database Driver & Version: MySQL 8.0.13
- monolog/monolog: 1.23.0 and 1.24.0 too

### Description:
When the code throws out the event, in the console I see an error processing it:

```
ErrorException  : Expected array for frame 0

  at /home/vagrant/code/example.com/vendor/monolog/monolog/src/Monolog/Formatter/LineFormatter.php:144
    140|         }
    141| 
    142|         $str = '[object] ('.get_class($e).'(code: '.$e->getCode().'): '.$e->getMessage().' at '.$e->getFile().':'.$e->getLine().$previousText.')';
    143|         if ($this->includeStacktraces) {
  > 144|             $str .= "\n[stacktrace]\n".$e->getTraceAsString()."\n";
    145|         }
    146| 
    147|         return $str;
    148|     }

  Exception trace:

  1   Exception::getTraceAsString()
      /home/vagrant/code/example.com/vendor/monolog/monolog/src/Monolog/Formatter/LineFormatter.php:146

  2   Monolog\Formatter\LineFormatter::normalizeException(Object(ErrorException))
      /home/vagrant/code/example.com/vendor/monolog/monolog/src/Monolog/Formatter/NormalizerFormatter.php:95

  Please use the argument -v to see more details.
```

Test code:
```php
Artisan::command('foo:bar', function () {
    $foo = $bar;
});
```

Error handling of exceptions in the console occurs whenever there is no catching of exceptions.

![2018-11-05_20-46-09](https://user-images.githubusercontent.com/10347617/48016171-f3899400-e13b-11e8-8d61-bba99a79442c.png)
If you comment out the 146 line, then the monolog returns a real error, not its own.

Is this an error generated by the framework or an error in the package processed by it?

By the way, calling method `getTrace()` does not cause errors and returns an array. But calling the conversion method on a string will certainly cause an error.

Command `dd($e->getTraceAsString());` also causes an error.

**Comment by driesvints** [comment]
With the above test code I get the following exception: 

```
$ ~/Sites/test-app  a foo:bar

   ErrorException  : Undefined variable: bar

  at /Users/driesvints/Sites/test-app/app/Providers/AppServiceProvider.php:17
    13|      */
    14|     public function boot()
    15|     {
    16|         \Artisan::command('foo:bar', function () {
  > 17|             $foo = $bar;
    18|         });
    19|     }
    20| 
    21|     /**

  Exception trace:

  1   Illuminate\Foundation\Bootstrap\HandleExceptions::handleError("Undefined variable: bar", "/Users/driesvints/Sites/test-app/app/Providers/AppServiceProvider.php", [])
      /Users/driesvints/Sites/test-app/app/Providers/AppServiceProvider.php:17

  2   Illuminate\Foundation\Console\ClosureCommand::App\Providers\{closure}()
      /Users/driesvints/Sites/test-app/vendor/laravel/framework/src/Illuminate/Container/BoundMethod.php:29

  Please use the argument -v to see more details.
```

I'll need more info and/or code to debug this further. Please post relevant code like jobs, commands, notifications, events, listeners, controller methods, routes, etc. You may use https://paste.laravel.io to post larger snippets or just reply with shorter code snippets. Thanks!

**Comment by andrey-helldar** [comment]
This is both funny and not. I found a problem.
My project uses a package that is responsible for saving the exceptions to the database, then creating an application in jira and notifying in slack. This error was generated by the method of serialization of exceptions for writing. This is necessary in order to work with it as with the class of the executor during deserialization.
Why did the error occur in freshly installed applications? I first installed the entire package set ...
